src/olympia/abuse/tests/test_serializers.py

```diff
@@ -1,9 +1,9 @@
-from olympia.accounts.serializers import BaseUserSerializer
 from olympia.abuse.models import AbuseReport
 from olympia.abuse.serializers import (
-    AddonAbuseReportSerializer, UserAbuseReportSerializer)
-from olympia.amo.tests import (
-    addon_factory, BaseTestCase, user_factory)
+    AddonAbuseReportSerializer, UserAbuseReportSerializer
+)
```

Comment:
It'd be great if we could keep the `)` on the end of the line.  Mainly because I prefer it that way but also so less changes in this patch.